bindings/matrix-sdk-crypto-js/src/lib.rs

```diff
@@ -40,6 +40,27 @@ pub mod vodozemac;
 
 use wasm_bindgen::prelude::*;
 
+/// Object containing the versions of the Rust libraries we are using.
+#[wasm_bindgen(getter_with_clone)]
+#[derive(Debug)]
+pub struct Versions {
+    /// The version of the vodozemac crate.
+    #[wasm_bindgen(readonly)]
+    pub vodozemac: &'static str,
```

Comment:
If you convert the `&str` to `JsString`, then no conversion will be applied by wasm-bindgen, or basically from Wasm to JS.

Reply:
Fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/1664/commits/b890cfbf37bc503b19353691d11c38aef17c2069.